src/Components/Publishing/Display/DisplayAd.tsx

```diff
@@ -72,11 +74,11 @@ export const DisplayAd: SFC<DisplayAdProps> = props => {
 }
 
 const DisplayAdContainer = styled(Flex)<DisplayAdContainerProps>`
-  margin: ${props => (props.isStandard ? "0" : "0 auto")};
-  border-top: ${props =>
-    props.isSeries ? `1px solid ${color("black10")}` : "none"};
-  background: ${props =>
-    props.isSeries ? color("black100") : color("black5")};
+  margin: ${p => (p.isStandard ? "0" : "0 auto")};
+  border-top: ${p => (p.isSeries ? `1px solid ${color("black10")}` : "none")};
+  background: ${p => (p.isSeries ? color("black100") : color("black5"))};
   text-align: center;
   width: 100%;
+  visibility: ${p =>
```

Comment:
why didn't you use `display: none` which collapses the height instead of `visibility: hidden`?

Reply:
B/C it wasn't rendering, I don't know why.